### ISSUE TITLE
Environmental qualities

### ISSUE THREAD

**Issue by pbuttigieg** [gkoutos/pato#53]
Greetings PATO Team,

Over at the [Environment Ontology](https://github.com/EnvironmentOntology/envo), we (myself and @cmungall) were considering the creation of some experimental quality-like classes that seem within the PATO domain:
e.g. "aquatic", "marine", "littoral", "deciduous", etc

Draft example definition...
> "marine": A quality that inheres in an environmental system (the bearer) by virtue of that bearer being   > determined by an ocean or sea.
Where "determined by" is an ENVO relation expressing strong causal influence 

The qualities of interest are generally used in the context of describing environments and ecosystems, thus we would be happy to develop them in ENVO as "environmental qualities"; however, we think that these should eventually merge into PATO once the experimental phase is over. Would we have a green light on this?

**Comment by cmungall** [comment]
We have an analogous situation with anatomy. We're defining qualities like 'collagenous' as equivalentTo

    'inheres in' some ('composed primarily of' some collagen)

This effectively makes them syntactic sugar. You could write the same thing without PATO. But it's useful as various apps 'expect' to have a Q to place in certain fields so it's continuous.

The modularization is slightly odd as we have some stuff over here in PATO in an odd bridge file, other stuff in uberon. In some ways it would easiest to organize this all over here in uberon or caro, minimize inter-module dependencies. But there is a certain historic attachment to having any kind of anatomical quality in PATO.

There isn't really any such expectation for qualities that are purely environmental (of course there are qualities like 'liquid' that are being used for multiple domains). For a purely environmental quality it may be simpler to minimize inter-module crossings and keep these in ENVO. Hard to say what is purely environmental. With my planteome hat on I can see wanting to use deciduous as a trait descriptor.

**Comment by pbuttigieg** [comment]
Yes, the 'problem' with environments is that pretty much any material entity can be 'hubs'  of some environmental system (and thus realise the environmental feature 'role'). I don't think we can really avoid modularity and importing a bunch of ontologies (plants, qualities, etc). Things like 'arid' and 'oceanic' can be considered closer to 'pure' environmental, but that's probably because there's no physical geography ontology interacting with ENVO.

Perhaps the model whereby 1) a set of potentially-out-of-domain classes are drafted in the ontology that's working with them given that 2) the respective domain ontology (here, PATO) is made aware of these and contributes to their definition so that 3) the classes will be surrendered en masse to the appropriate domain ontology (and given alternate IDs?). Is this feasible?
